src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -0,0 +1,91 @@
+package frc.robot.subsystems.intake;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import com.revrobotics.SparkMaxPIDController;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Ports;
+import frc.robot.subsystems.LoggedSubsystem;
+
+public class Intake extends LoggedSubsystem<IntakeLoggedInputs> {
+    public static Intake INSTANCE;
+    private final CANSparkMax superiorSparkMax = new CANSparkMax(Ports.Intake.SUPERIOR_SPARK_MAX_PORT, CANSparkMaxLowLevel.MotorType.kBrushless);
+    private final CANSparkMax inferiorSparkMax = new CANSparkMax(Ports.Intake.INFERIOR_SPARK_MAX_PORT, CANSparkMaxLowLevel.MotorType.kBrushless);
```

Comment:
Also, maybe find some different names. Even slave and master is more appealing.

Reply:
done.